packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md

```diff
@@ -1,8 +1,8 @@
-Access the external SAML Identity Provider (IdP) you want to integrate with and gather the following information. You use this information to configure the SAML Identity Provider in Okta in the next step. Make sure the SAML IdP supports Service Provider-initiated SAML.
+Access the external SAML IdP you want to integrate with and gather the following information. Use this information to configure the SAML IdP in Okta in the next step. Ensure the SAML IdP supports Service Provider-initiated SAML.
 
-> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can try using this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
+> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can use this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
 
-Sometimes the issuer, single sign-on URL, and certificate aren't available from the external IdP until the metadata is uploaded to the external IdP. This metadata includes the Assertion Consumer Service URL (ACS URL) and the Audience URI. And, the ACS URL and Audience URI values aren't available until the IdP in Okta is configured. If that's the case with your external IdP, continue to the next step.
+When you set up an external IdP, sometimes the **Issuer**, **Single Sign-On URL**, and **Certificate** values aren't available. This information may not be available until the metadata is uploaded to the IdP. Also, the ACS URL and Audience URI values aren't available until you configure the IdP in Okta. If that's the case with your external IdP, continue to the [next step](#create-an-idp-in-okta).
```

Comment:
I'm not sure if the user would "set up an external IdP". These instructions are for configuring an app (or registering an app) in the external IdP.

Can you move this information right after the "Use this information... " line?

Reply:
Rewording suggestion:
```suggestion
If you don't have the issuer URI, SSO URL, and certificate values, continue to the [next step](#create-an-idp-in-okta) to generate the SAML metadata. The external IdP requires the ACS URI and audience URI, which are included in the generated metadata. Upload the metadata to your external IdP to obtain the issuer, single sign-on URL, and certificate values.
```